pulsar-client-reactive-api/src/main/java/org/apache/pulsar/reactive/client/api/ReactiveMessageSenderSpec.java

```diff
@@ -27,60 +27,202 @@
+	 * @see ProducerBuilder#batchingMaxPublishDelay
+	 */
 	Duration getBatchingMaxPublishDelay();
 
+	/**
+	 * Gets the partition switch frequency while batching of messages is enabled and using
+	 * round-robin routing mode for non-keyed message.
+	 * @return the frequency of partition switch
+	 * @see ProducerBuilder#roundRobinRouterBatchingPartitionSwitchFrequency
+	 */
 	Integer getRoundRobinRouterBatchingPartitionSwitchFrequency();
 
+	/**
+	 * Gets the maximum number of messages permitted in a batch.
+	 * @return the maximum number of messages in a batch
+	 * @see ProducerBuilder#batchingMaxMessages
+	 */
 	Integer getBatchingMaxMessages();
 
+	/**
+	 * Gets the maximum number of bytes permitted in a batch.
+	 * @return the maximum number of messages in a batch
+	 * @see ProducerBuilder#batchingMaxBytes
+	 */
 	Integer getBatchingMaxBytes();
 
+	/**
+	 * Gets whether automatic batching of messages is enabled for the producer.
+	 * @return true if batching is enabled
+	 * @see ProducerBuilder#enableBatching
+	 */
 	Boolean getBatchingEnabled();
 
+	/**
+	 * Gets the batcher builder of the producer.
+	 * @return the batcher builder
+	 * @see ProducerBuilder#batcherBuilder
+	 */
 	BatcherBuilder getBatcherBuilder();
 
+	/**
+	 * Gets whether chunking of messages is enabled.
+	 * @return true if message chunking is enabled
+	 * @see ProducerBuilder#enableChunking
+	 */
 	Boolean getChunkingEnabled();
 
+	/**
+	 * Gets the key reader to be used to encrypt the message payloads.
+	 * @return the key reader to be used to encrypt the message payloads
+	 * @see ProducerBuilder#cryptoKeyReader
+	 */
 	CryptoKeyReader getCryptoKeyReader();
 
+	/**
+	 * Gets the public encryption key names, used by producer to encrypt the data key.
+	 * @return the public encryption key names
+	 * @see ProducerBuilder#addEncryptionKey
+	 */
 	Set<String> getEncryptionKeys();
 
+	/**
+	 * Gets the compression type for the producer.
+	 * @return the compression type
+	 * @see ProducerBuilder#compressionType
+	 */
 	CompressionType getCompressionType();
 
+	/**
+	 * Gets the baseline for the sequence ids for messages published by the producer.
+	 * @return the initial sequence id
+	 * @see ProducerBuilder#initialSequenceId
+	 */
 	Long getInitialSequenceId();
 
+	/**
+	 * Gets whether partitioned producer will automatically discover new partitions at
+	 * runtime.
+	 * @return true if auto discovery of the partition configuration changes is enabled
+	 * @see ProducerBuilder#autoUpdatePartitions
+	 */
 	Boolean getAutoUpdatePartitions();
 
+	/**
+	 * Gets the interval of partitions updates if autoUpdatePartitions is enabled.
+	 * @return the interval of partitions updates
+	 * @see ProducerBuilder#autoUpdatePartitionsInterval
+	 */
 	Duration getAutoUpdatePartitionsInterval();
 
+	/**
+	 * Gets whether the multiple schema mode is enabled.
+	 * @return true if the multiple schema mode is enabled
+	 * @see ProducerBuilder#enableMultiSchema
+	 */
 	Boolean getMultiSchema();
```

Comment:
btw. We don't currently have support for multi-schema. I created #26 to address this.